### ISSUE TITLE
I can't install fink on Mac OS 13.14

### ISSUE THREAD

**Issue by zbc1508** [fink/fink#248]
Error message(terminal window):

Fink must be installed and run with superuser (root)
privileges. Fink can automatically try to become root
when it's run from a user account. Since you're
currently running this script as a normal user, the
method you choose will also be used immediately for this
script. Available methods:

(1)	Use sudo
(2)	Use su
(3)	None, fink must be run as root

Choose a method: [1] 1

Checking package... looks good (fink-0.45.99.git).
Checking system... arm-apple-darwin22.5.0
13.4 does not match the expected value of 10.18. Please run `fink selfupdate` to download a newer version of fink at /Downloads/fink-master/perlmod/Fink/Services.pm line 1430.

where can I download the right version of fink?

**Comment by nieder** [comment]
Make sure you've cloned the version of Fink from this fork:
```
git clone https://github.com/TheSin-/fink fink-thesin
cd fink-thesin
git checkout dpkg1.16
./bootstrap
```

**Comment by zbc1508** [comment]
(base) abc~ % git clone https://github.com/TheSin-/fink fink-thesin
Cloning into 'fink-thesin'...
fatal: unable to access 'https://github.com/TheSin-/fink/': Recv failure: Operation timed out

This page is 404。。。

**Comment by nieder** [comment]
Works here:

<img width="403" alt="image" src="https://github.com/fink/fink/assets/1105137/268cf150-7b0d-447f-b625-957d9a1b251c">

```
$ git clone https://github.com/TheSin-/fink fink-thesin
Cloning into 'fink-thesin'...
remote: Enumerating objects: 33768, done.
remote: Counting objects: 100% (726/726), done.
remote: Compressing objects: 100% (363/363), done.
remote: Total 33768 (delta 413), reused 653 (delta 362), pack-reused 33042
Receiving objects: 100% (33768/33768), 10.99 MiB | 3.63 MiB/s, done.
Resolving deltas: 100% (22262/22262), done.
```

**Comment by zbc1508** [comment]
It works! Thanks a lot!

**Comment by cooljeanius** [comment]
> Make sure you've cloned the version of Fink from this fork:
> 
> ```
> git clone https://github.com/TheSin-/fink fink-thesin
> cd fink-thesin
> git checkout dpkg1.16
> ./bootstrap
> ```

Sure would be nice to get PR #66 merged after all these years...

**Comment by dmacks** [comment]
OP was solved via TheSin's fork, and #66 is now merged into master. Looks like we're done here.
